### ISSUE TITLE
Easy way to attach custom errors to schemas

### ISSUE THREAD

**Issue by ikitommi** [metosin/malli#254]
Maybe lift the ErrorSchema protocol from malli.error to malli.core to enable setting the custom errors like this:

```clj
(m/-simple-schema {:type :symbol, :pred symbol?, :error/message {:en "should be symbol"}}})
```

**Comment by ikitommi** [comment]
This will be fixed in #255 .

**Comment by ikitommi** [comment]
```clj
(-> (m/-simple-schema
      {:type :symbol
       :pred symbol?
       :type-properties {:error/message {:en "should be symbol"}}})
    (m/explain "invalid")
    (me/humanize))
; => ["should be symbol"]
```
